Document notification center (#456)
discord-userdoccers/discord-userdoccers: pages/topics/gateway-events.mdx

| [Message Reaction Remove Emoji](#message-reaction-remove-emoji)                             | All reactions for a given emoji were explicitly removed from a message                         |
| [Recent Mention Delete](#recent-mention-delete)                                             | Recent message that mentioned the current user was acknowledged                                |
| [Last Messages](#last-messages)                                                             | Response to [Request Last Messages](#request-last-messages)                                    |
| [Notification Center Item Create](#notification-center-item-create)                         | Notification center item was created                                                           |
| [Notification Center Item Delete](#notification-center-item-delete)                         | Notification center item was deleted                                                           |
| [Notification Center Items Ack](#notification-center-items-ack)                             | Notification center item was acknowledged                                                      |

> | [Notification Center Item Completed](#notification-center-item-completed)                   | Notification center items of certain item enum were completed                                  |

Reviewer: ```suggestion
| [Notification Center Item Completed](#notification-center-item-completed)                   | Notification center item of a certain action type was completed                                  |
```

| messages | array[[message](/resources/message#message-object) object] | Last messages of the requested channels |

### Notification Center

#### Notification Center Item Create

Sent when a notification center item is created. The inner payload is a [notification center item](/resources/notification-center#notification-center-item-object) object.

#### Notification Center Item Delete

Sent when a notification center item is deleted.

###### Notification Center Item Delete Structure

| Field   | Type      | Description                                    |
| ------- | --------- | ---------------------------------------------- |
| id      | snowflake | The ID of the deleted notification center item |
| user_id | snowflake | The ID of the current user                     |

#### Notification Center Items Ack

Sent when a notification center item is acknowledged.

###### Notification Center Items Ack Structure

| Field | Type      | Description                                         |
| ----- | --------- | --------------------------------------------------- |
| id    | snowflake | The ID of the acknowledged notification center item |

#### Notification Center Item Completed

> Sent when a notification center items of certain item enum are completed.

Reviewer: ```suggestion
Sent when a notification center action set is completed.
```
